recipes/jsdom/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "11.0.0" %}
+{% set node_version = os.environ.get('NODEJS_VERSION') or '6.*' %}
+
+package:
+  name: jsdom
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/tmpvar/jsdom/archive/{{ version }}.tar.gz
+  sha256: 9fb69879902bde82c9d7be1be3c0d1c16ac00190326ac2fdb4e64f01b590c2e8
+
+build:
+  number: 1
+  script: npm install . && npm install -g .
```

Comment:
Yes, I do not now the exact reason why it is like this, but the global install only does not find/install the dependencies correct for some reason in conda-build.

It doesn't change the package size, so that shouldn't be a big downside.